src/streamlink/plugins/telefe.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+from streamlink.plugin import Plugin
+from streamlink.plugin.api import http, useragents, validate
+from streamlink.stream import HLSStream
+
+
+class Telefe(Plugin):
+    _url_re = re.compile(r'https?://telefe.com/.+')
+
+    @classmethod
+    def can_handle_url(cls, url):
+        return Telefe._url_re.match(url)
+
+    def _get_streams(self):
+        TELEFE_UA = "Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.36 OPR/44.0.2510.1218"
+        res = http.get(self.url, headers={'User-Agent': TELEFE_UA})
```

Comment:
You can use `{'User-Agent': useragents.CHROME}` instead, same goes for the other `http.get`.